airflow/migrations/versions/849da589634d_prefix_dag_permissions.py

```diff
@@ -167,4 +233,8 @@ def upgrade():
 
 
 def downgrade():
-    pass
+    db = SQLA()
+    db.session = settings.Session
+    undo_migrate_to_new_dag_permissions(db)
+    db.session.commit()
```

Comment:
This commit seems unnecessary?

Reply:
Aos from what I can tell `undo_migrate_to_new_dag_permissions` only uses `db.session`; why not just pass that into the function, but the entire `db` object?